DXRModules/DeusEx/Classes/DXRFixup.uc

```diff
@@ -1118,3 +1118,36 @@ function SetAllLampsState(optional bool type1, optional bool type2, optional boo
     }
 #endif
 }
+
+static function DXRFixup GetCurrentFixupModule(DXRando dxr)
+{
+    switch (dxr.dxInfo.missionNumber) {
+        case 0:
+            return DXRFixup(dxr.FindModule(class'DXRFixupM00', true));
```

Comment:
You can just do it by the base class, DXRFixups, also you can do class'DXRFixups'.static.Find()